controllers/main.go

```diff
@@ -1210,16 +1214,24 @@ func (controller *MainController) Index(c web.C, r *http.Request) (string, int)
 	t := controller.GetTemplate(c)
 
 	// execute the named template with data in c.Env
-	widgets := helpers.Parse(t, "home", c.Env)
-
+	widgets, err := helpers.Parse(t, "home", c.Env)
+	if err != nil {
+		log.Infof("helpers.Parse: home failed: %v", err)
+		return "/error?r=/stats", http.StatusSeeOther
```

Comment:
We removed the functionality to redirect from the error page. Param `?r=/stats` is not used anywhere and can be removed.

I notice this is present in a few other places, not related to your PR. Would you mind removing them please?